src/bin/ion/commands/mod.rs

```diff
@@ -70,10 +71,6 @@ pub trait IonCliCommand {
         }
     }
 
-    /// Sets up the pager (e.g. `less`)  to which long text output will be directed. The default
-    /// implementation does not configure a pager.
-    fn set_up_pager(&self) {}
-
```

Comment:
What's happening here?

Reply:
This was added to the `IonCliCommand` trait, but the only command using it was `inspect`. Each place that it's implemented needed a `#[cfg(not(windows))]`, which was 2-3 places. I decided to remove it from the trait so it would only need one instead. If/when other commands need it it'll be easy to add back.